src/GRA.Controllers/ViewModel/MissionControl/Programs/ProgramDetailViewModel.cs

```diff
@@ -15,6 +16,8 @@ public class ProgramDetailViewModel
         [Range(0, 2)]
         public int SchoolValues { get; set; }
 
+        [DisplayName("Badge Alt-Text")]
+        public string BadgeAltText { get; set; }
```

Comment:
Property should have the MaxLength attribute to help check for ModelErrors.

Reply:
Fixed in commit d65c7ad1f42bf397ccad695a33bd75da81206656